modules/t/test-genome-DBs/parus_major1/core/external_db.txt

```diff
@@ -428,11 +424,11 @@ The main requirement behind this entry, is to be able to link TAIR GO annotation
 50842	Ensembl_Plants	\N	XREF	0	Ensembl Plants	MISC	\N	\N	Maping from translation to XREF
 50843	MaizeGDB	\N	XREF	0	MaizeGDB	MISC	\N	\N	MaizeGDB is a community-oriented, long-term, federally funded informatics service to researchers focused on the crop plant and model organism Zea mays.
 50845	Botrytis_community_symbol	\N	KNOWN	50	Botrytis community symbol	MISC	\N	\N	\N
-50846	BARLEX	\N	KNOWN	1	BARLEX: The Barley Genome Explorer 	MISC	\N	\N	The <A HREF="http://barlex.barleysequence.org/">barley genome explorer</A> allows a visual inspection of BAC overlaps. In addition to a graph view the user can access dotplots and fingerprints to compare two individual BACs with each other. Information to genes and markers is available as well. 
+50846	BARLEX	\N	KNOWN	1	BARLEX: The Barley Genome Explorer 	MISC	\N	\N	The <A HREF="http://barlex.barleysequence.org/">barley genome explorer</A> allows a visual inspection of BAC overlaps. In addition to a graph view the user can access dotplots and fingerprints to compare two individual BACs with each other. Information to genes and markers is available as well.
 50847	Ngyuen_et_al_2012	\N	PRED	0	PMID 23153364	MISC	\N	\N	The complete swine olfactory subgenome: expansion of the olfactory gene repertoire in the pig genome. BMC Genomics201213:584 PMID: 23153364
 50848	Plant_Reactome_Reaction	\N	KNOWNXREF	0	Plant Reactome Reaction ID	MISC	\N	\N	Plant Reactome is a database of plant metabolic and regulatory pathways maintained by Gramene. This ID links to a Reaction in Plant Reactome.
 50849	Plant_Reactome_Pathway	\N	KNOWNXREF	0	Plant Reactome Pathway ID	MISC	\N	\N	Plant Reactome is a database of plant metabolic and regulatory pathways maintained by Gramene. This ID links to a Pathway in Plant Reactome.
-50850	CGSNL	\N	KNOWNXREF	0	Committee on Gene Symbolization, Nomenclature and Linkage of the Rice Genetics Cooperative	MISC	\N	\N	The Committee on Gene Symbolization, Nomenclature and Linkage (CGSNL) of the Rice Genetics Cooperative has revised the gene nomenclature system for rice (Oryza) to take advantage of the completion of the rice genome sequence and the emergence of new methods for detecting, characterizing, and describing genes in the biological community (<A HREF="https://doi.org/10.1007/s12284-008-9004-9">McCouch 2008</A>).
```

Comment:
The proposed change applied to this DOI breaks the link. Please can the DOI link be kept as it currently is, i.e. "https://doi.org/10.1007/s12284-008-9004-9"